### PR TITLE
cmd/tsuru: unify service-info and service-doc

### PR DESCRIPTION
**Pull request by lucasweiblen**
#1152 
that's it for the tsuru part of this issue, right? If there's something missing or wrong please point me in the right direction.
I will open a PR for the tsuru-client part as well in the next days.

### CODE REVIEW

File: service/service.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ type Service struct {
 	Teams        []string
 	Doc          string
 	IsRestricted bool `bson:"is_restricted"`
+	Instances    []ServiceInstance
```
Comment (andrewsmedina) [inline service/service.go]: I'm -1 for to add this field.

I believe that the better approach is to add a `service.FindServiceInstanceByName("name", &user)` method and to split this handler in two handlers, one to get the service info and another to get service instances by service name.

Comment (lucasweiblen) [inline service/service.go]: cool, Andrews. I agree it's a better approach. :smile: 
Let's see if the guys have any other suggestions and I can start working on improving this PR.

Thanks for your feedback.

Comment (fsouza) [inline service/service.go]: I don't think there's much else to review. With Andrews' suggestion, most changes move to the client, that will now need to issue to API calls and combine the results.

So, you will:

1. remove the service doc handler (or convert it into a handler that returns full information of the service)
1. add a handler for listing instances of a service (that the user has access to, obviously)

And then comes the client changes.

Thank you!

Comment (lucasweiblen) [inline service/service.go]: thanks @fsouza.. will work on these improvements :smile:

**Comment by fsouza** [comment]
I will close this pull request, as @raulgbrmf is currently working on an alternative solution.

@lucasweiblen thanks for contributing!

**Pull request closed without merging**